### DISCUSSION TITLE
Retrieve results

### DISCUSSION THREAD

**Discussion by ltgustavsen** [Mattk70/Chirpity-Electron#132] [Q&A] [answered]
First, I will say this software is really awesome. I have just started out with a wildlife acoustic song meter micro and the plan is to use it for localizing birds. 

In the video there is a retrieve results options under analyze. I cannot find that option on my Mac.   Or maybe there is another option to retrieve results from a folder of audio files. If not something like Kaleidoscope that can read detections from a csv file would be nice.

**Comment by ltgustavsen** [comment]
<img width="847" alt="retrvie results missing" src="https://github.com/Mattk70/Chirpity-Electron/assets/15889329/a20b8229-795f-474e-bf4b-1833d0a2e115">

**Reply by Mattk70** [reply, marked as answer]
Hi @ltgustavsen, you will need to save your results before you can retrieve them. Records > Save to Archive (Ctrl or ⌘ s)

**Reply by ltgustavsen** [reply]
Thanks, 
Sorry I missed that. It works very well.
